### PR TITLE
New Exercise: ISBN-Verifier

### PR DESCRIPTION
**Pull request by stfnsr**
I saw on http://exercism.io/languages/elixir/todo that there's a new problem-specification which apparently no language implemented yet.

Looking forward to a rigorous review :)

### CODE REVIEW

File: exercises/isbn-verifier/isbn_verifier_test.exs [hunk 1]
```diff
@@ -0,0 +1,74 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("isbn_verifier.exs", __DIR__)
+end
+
+ExUnit.start()
+ExUnit.configure(exclude: :pending, trace: true)
+
+defmodule ISBNVerifierTest do
+  use ExUnit.Case
+
```
Comment (devonestes) [inline exercises/isbn-verifier/isbn_verifier_test.exs]: Could you please leave the `@tag :pending` here, but commented out? At this point it's mostly for consistency's sake, but it's also helpful for folks to see sometimes.

File: exercises/isbn-verifier/example.exs [hunk 2]
```diff
@@ -0,0 +1,36 @@
+defmodule ISBNVerifier do
+  @doc """
+    Checks if a string is a valid ISBN-10 identifier
+
+    ## Examples
+
+      iex> ISBNVerifier.isbn?("3-598-21507-X")
```
Comment (devonestes) [inline exercises/isbn-verifier/example.exs]: Great doctests, thanks!

File: exercises/isbn-verifier/isbn_verifier_test.exs (hunk 1 above)
Comment (stfnsr) [reply exercises/isbn-verifier/isbn_verifier_test.exs]: Got it :)

File: exercises/isbn-verifier/README.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+* Generate valid ISBN, maybe even from a given starting ISBN
+
+
+## Running tests
+
+Execute the tests with:
+
+```bash
+$ elixir bob_test.exs
```
[the remark this answers is not in the document]
Comment (stfnsr) [reply exercises/isbn-verifier/README.md]: This is new, right?
Because all(?) other exercise readmes do have this line with `bob_test.exs` in them.

**Pull request merged**